src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -143,9 +148,16 @@ public Client toModelType() throws IllegalValueException {
         final ContractExpiryDate modelContractExpiryContractExpiryDate =
                 ParserUtil.parseContractExpiryDate(contractExpiryDate);
 
+        LastModifiedInstant modelLastModifiedInstant;
+        if (lastModifiedInstant == null) {
+            modelLastModifiedInstant = new LastModifiedInstant();
+        } else {
+            modelLastModifiedInstant = new LastModifiedInstant(lastModifiedInstant);
+        }
```

Comment:
When will this be null? Should it just throw an exception when null if its not supposed to be null?

Reply:
It can only be null if it doesn't exist in storage, but I feel like the corruption of this data field is too insignificant to have to throw away all existing data on this client.

Reply:
ok shld add a comment then